.github/release-drafter.yaml

```diff
@@ -11,6 +11,11 @@ categories:
     label: documentation
 tag-template: - $TITLE @$AUTHOR (#$NUMBER)
 template: |
+  ## [Docker image](https://quay.io/repository/signalfx/splunk-otel-collector)
```

Comment:
Would you be opposed to putting this after the changes so it's just above the other artifacts?  Definitely helpful to have this direct link in general but if breaking changes are introduced it be helpful to have the link after the disclaimers.

Reply:
No, that would work too. I’ll make the change.